doc/dataset.md

```diff
@@ -194,13 +194,28 @@ idealization {math}`( \bar{\mathbf{a}}_\mathrm{s} \;
 Cartesian coordinates of the given input unit cell. The rotation
 matrix {math}`\boldsymbol{R}` is defined by
 ```{math}
-   ( \bar{\mathbf{a}}_\mathrm{s} \;
-   \bar{\mathbf{b}}_\mathrm{s} \; \bar{\mathbf{c}}_\mathrm{s} )
-   = ( \boldsymbol{R} \mathbf{a}_\mathrm{s} \;
-   \boldsymbol{R} \mathbf{b}_\mathrm{s} \; \boldsymbol{R}
-   \mathbf{c}_\mathrm{s} ).
+:label: rotation-in-standardization
+
+( \bar{\mathbf{a}}_\mathrm{s} \;
+\bar{\mathbf{b}}_\mathrm{s} \; \bar{\mathbf{c}}_\mathrm{s} )
+= ( \boldsymbol{R} \mathbf{a}_\mathrm{s} \;
+\boldsymbol{R} \mathbf{b}_\mathrm{s} \; \boldsymbol{R}
+\mathbf{c}_\mathrm{s} ).
 ```
 
+More precisely, this rotation matrix is an orthonormal matrix. Since {math}`(
+\mathbf{a}_\mathrm{s} \; \mathbf{b}_\mathrm{s} \; \mathbf{c}_\mathrm{s} )` can
+be distored, to make {math}`\boldsymbol{R}` orthonormal, it is calculated as
+follows. Make cubes of {math}`( \mathbf{a}_\mathrm{s} \; \mathbf{b}_\mathrm{s}
+\; \mathbf{c}_\mathrm{s} )` and {math}`( \bar{\mathbf{a}}_\mathrm{s} \;
+\bar{\mathbf{b}}_\mathrm{s} \; \bar{\mathbf{c}}_\mathrm{s} )` by
+{math}`\mathrm{L} = \begin{pmatrix}\frac{\mathbf{a}}{|\mathbf{a}|} &
```

Comment:
$\mathbf{c} \times \mathbf{a}$ is replaced with $(\mathbf{a} \times \mathbf{b}) \times \mathbf{a}$ instead?
https://github.com/spglib/spglib/blob/5e98f3241e9d429f0f9168d7f30af4daa551058b/src/refinement.c#L1546-L1547

Reply:
You are correct!